src/main/java/net/pms/configuration/RendererConfiguration.java

```diff
@@ -561,11 +561,12 @@ public boolean associateIP(InetAddress sa) {
 	}
 
 	public static void calculateAllSpeeds() {
-		for (InetAddress sa : addressAssociation.keySet()) {
+		for (Entry<InetAddress, RendererConfiguration> key : addressAssociation.entrySet()) {
+			InetAddress sa = key.getKey();
 			if (sa.isLoopbackAddress() || sa.isAnyLocalAddress()) {
 				continue;
 			}
-			RendererConfiguration r = addressAssociation.get(sa);
+			RendererConfiguration r = key.getValue();
```

Comment:
I'd say it's very confusing that the entry is called "key"

Reply:
That's a good point, it would be better to use more customized terms

Reply:
Thanks for comments. Done